tests/test_main.py

```diff
@@ -83,32 +82,25 @@ def test_close_duplicates(self,
         """
         # Set up return values
         mock_load_config.return_value = self.mock_config
-        mock_u.pagure_issues.return_value = ['mock_issue_github']
-        mock_u.github_issues.return_value = ['mock_issue_pagure']
+        mock_u.github_issues.return_value = ['mock_issue_github']
```

Comment:
Were the return values in the old code _supposed_ to be reversed, or was that a bug that you've corrected here??

Reply:
I didn't test the old code to confirm, but I did notice that and assumed it was a oversight.